src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityServiceTest.kt

```diff
@@ -104,10 +104,21 @@ class EligibilityServiceTest {
   }
 
   @Test
-  fun `Person is on ineligible EDS - has a APD with a PED in the past - not eligible for CVL `() {
+  fun `Person is on ineligible EDS - has a APD with a PED today - not eligible for CVL `() {
     val result = service.isEligibleForCvl(
       aPrisonerSearchResult.copy(
-        paroleEligibilityDate = LocalDate.now(clock).minusDays(1),
+        paroleEligibilityDate = LocalDate.now(clock),
+        actualParoleDate = LocalDate.now(clock),
+      ),
+    )
+    assertThat(result).isFalse()
+  }
+
+  @Test
+  fun `Person is on ineligible EDS - has a APD with a PED in the future - not eligible for CVL `() {
```

Comment:
No harm in having this, but I imagine it's actually return false on the future PED check earlier in the `isEligibleForCvl` check
Might be worth keeping regardless as it will protect against that PED check changing in future